vue-frontend/src/components/jobs-thankyou/ContributionSection.vue

```diff
@@ -0,0 +1,243 @@
+<template>
+  <div class="xll:tw-container tw-my-20 lg:tw-mb-40">
```

Comment:
Still not looking properly. Title should align with other divs as shown in image

![Untitled design (1)](https://github.com/canopas/canopas-website/assets/69897605/7003af07-af94-4f0d-95f2-9325dd377a47)

Reply:
In figma it is like this only... little bit outer side from perks and benefit section , not aligned in same line .

Reply:
But its not same for all width. Can you check for all widths?